Gemfile

```diff
@@ -44,19 +43,19 @@ gem "sidekiq"
 gem "devise-i18n"
 gem "i18n-js", ">= 3.0.0.rc11"
 gem "rails-i18n"
-gem "route_translator", ">=5.5.3"
+gem "route_translator", "~> 10.0"
 
 # Model hierarchical data
-gem "closure_tree"
+gem "closure_tree", "~> 7.3"
 
 # Tables
 gem "font-awesome-sass", "~> 5.12"
 gem "jquery-ui-rails"
 
-# We are using this fork because:
-# 1. 'tag_options' no longer valid in rails 5
-# 2. AR 'size' method triggers the count query which causes errors in rails 5
-gem "wice_grid", git: "https://github.com/sassafrastech/wice_grid.git", branch: "rails5"
+# We are using this fork because the original gem is no longer maintained. This fork seems to be where
```

Comment:
thanks for looking at all the forks & finding the best one for us to use!

Reply:
You're welcome!